src/Packages/Passport/Runtime/ThirdParty/Gree/Assets/Plugins/WebGLTemplates/unity-webview/unity-webview.js

```diff
@@ -0,0 +1,97 @@
+var unityWebView =
+    {
+        loaded: [],
+
+        init: function(name) {
+            $containers = $('.webviewContainer');
+            if ($containers.length === 0) {
+                $('<div style="position: absolute; left: 0px; width: 100%; height: 100%; top: 0px; pointer-events: none;"><div class="webviewContainer" style="overflow: hidden; position: relative; width: 100%; height: 100%; z-index: 1;"></div></div>')
+                    .appendTo($('#unity-container'));
+            }
+            var $last = $('.webviewContainer:last');
+            var clonedTop = parseInt($last.css('top')) - 100;
+            var $clone = $last.clone().insertAfter($last).css('top', clonedTop + '%');
+            var $iframe =
+                $('<iframe style="position:relative; width:100%; height100%; border-style:none; display:none; pointer-events:auto;"></iframe>')
+                    .attr('id', 'webview_' + name)
+                    .appendTo($last)
+            $iframe.on('load', function () {
+                $(this).attr('loaded', 'true');
+
+                var js = `window.Unity = {
+                            call:function(msg) {
+                                        parent.unityWebView.sendMessage('${name}', msg);
+                                    }
+                            };`;
+
+                this.contentWindow.eval(js);
+
+                window.addEventListener('message', function(event) {
+                    if (event.data.type === 'callback' && event.isTrusted) {
+                        unityInstance.SendMessage(name, 'CallOnAuth', event.data.url);
+                    }
+                }, false);
+            });
+            return $iframe;
+        },
+
+        sendMessage: function (name, message) {
+            unityInstance.SendMessage(name, "CallFromJS", message);
+        },
+
+        setMargins: function (name, left, top, right, bottom) {
```

Comment:
Can we remove `setMargins`/`setVisibility`?

I'm asking because when I was forking Voltstro, I tried to remove as many unnecessary functions as I could without breaking anything. However, some unused functions are still present as they're in the precompiled dll files, so it's fine if this needs to stay, too.